packages/api-demo/package.json

```diff
@@ -6,7 +6,7 @@
   "private": true,
   "scripts": {
     "serve:no-browser": "npm run serve -- --no-browser",
-    "serve": "live-server --mount=/:src --mount=/scripts/containerjs-api-bundle:node_modules/containerjs-api-bundle/build/containerjs-bundle.js --mount=/resources/notification.html:node_modules/containerjs-api-bundle/build/notification.html",
+    "serve": "live-server --mount=/:src --mount=/scripts/containerjs-api:node_modules/containerjs-api-browser/build/dist/containerjs-api.js --mount=/resources/notification.html:node_modules/containerjs-api-bundle/build/notification.html",
```

Comment:
Ah I missed that with it being off the side of my screen :laughing:. I think we still need it though (as OpenFin still needs it, even though we preload the JS). But it does need changing from bundle to openfin.